atdgen/src/oj_emit.ml

```diff
@@ -35,6 +35,7 @@ type param = {
 
 }
 
+let unopt = function None -> assert false | Some x -> x
```

Comment:
This is `Option.value_exn` in Import btw.

Reply:
Fixed, thanks. I might have brought this old code back when undoing the tag_field implementation.